src/class-rest.php

```diff
@@ -79,8 +79,17 @@ public function register_parsely_meta(): void {
 
 		$options     = $this->parsely->get_options();
 		$object_type = array_unique( array_merge( $options['track_post_types'], $options['track_page_types'] ) );
-		$args        = array( 'get_callback' => $callback );
 
+		/**
+		 * Filters the list of author object types that the Parse.ly REST API is hooked into.
+		 *
+		 * @since 3.1.0
+		 *
+		 * @param string[] $object_type Array of strings containing the object types, i.e. `page`, `post`, `term`.
+		 */
+		$object_type = apply_filters( 'wp_parsely_rest_object_type', $object_type );
```

Comment:
Let's make `$object_type` and the filter name plural, to indicate that we're expected an array or more than one type (at least by default).

Reply:
I did use singular because `register_rest_field` uses singular. Then again, in that case it can be a string or an array, in our case, if should always be an array.